.github/workflows/services-json.yml

```diff
@@ -0,0 +1,36 @@
+name: Services Validator
+
+on:
+  push:
+    paths-ignore:
+      - "cmake/**"
+    branches-ignore: [master]
+  pull_request:
+    paths:
+      - "plugins/rtmp-services/data/services.json"
+      - "plugins/rtmp-services/data/package.json"
+    branches-ignore: [master]
+
+jobs:
+  schema:
+    name: Schema
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: "recursive"
+      - name: Install & Configure Python
```

Comment:
Nitpick: for better readability I personally prefer having a blank line between steps